src/actions/adjust/Adjust.ts

```diff
@@ -215,10 +208,37 @@ function vibrance(strength?:number): EffectAction {
   return new EffectAction('vibrance', strength);
 }
 
+
+/**
+ * Maps an input color and those similar to the input color to corresponding shades of a specified output color, taking luminosity and chroma into account, in order to recolor an object in a natural way.</br>
+ * More highly saturated input colors usually give the best results. It is recommended to avoid input colors approaching white, black, or gray.
+ * @param {string} toColor
+ */
+function replaceColor(toColor:string): ReplaceColorAction {
+  return new ReplaceColorAction(toColor);
+}
+
+/**
+ * @description Converts the colors of every pixel in an image based on the supplied color matrix, in which the value of each color channel is calculated based on the values from all other channels (e.g. a 3x3 matrix for RGB, a 4x4 matrix for RGBA or CMYK, etc).</br>
+ * For every pixel in the image, take each color channel and adjust its value by the specified values of the matrix to get a new value.
+ * @param {number[][]} matrix
+ */
+function recolor(matrix:number[][]): RecolorAction {
```

Comment:
I know this is not in the spec but does it not look weird that recolor is not in camalcase?